ci/setup_env.sh

```diff
@@ -55,6 +55,7 @@ echo
 echo "update conda"
 conda config --set ssl_verify false
 conda config --set quiet true --set always_yes true --set changeps1 false
+conda install pip  # create conda to create a historical artifact for pip & setuptools
```

Comment:
this is from another commit. can you merge master

Reply:
Hmm, think something went wrong when I tried to rebase. Gone with a simple merge master now